tests/test_thunderbird.py

```diff
@@ -572,39 +825,50 @@ def quit_tb(tb):
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--tbname', help='Thunderbird executable name',
-        default='thunderbird')
+                        default='thunderbird')
     subparsers = parser.add_subparsers(dest='command')
     subparsers.add_parser('setup', help='setup Thunderbird for tests')
-    parser_send_receive = subparsers.add_parser('send_receive',
-        help='send and receive an email')
+    parser_send_receive = subparsers.add_parser(
+        'send_receive', help='send and receive an email')
     parser_send_receive.add_argument('--encrypted', action='store_true',
-        default=False)
+                                     default=False)
     parser_send_receive.add_argument('--signed', action='store_true',
-        default=False)
+                                     default=False)
     parser_send_receive.add_argument('--inline', action='store_true',
-        default=False)
+                                     default=False)
     parser_send_receive.add_argument('--with-attachment',
-        action='store_true', default=False)
+                                     action='store_true', default=False)
     args = parser.parse_args()
 
     # log only to stdout since logging to file have broken unicode support
     config.logDebugToFile = False
 
+    # get Thunderbird version
+    version = get_version(args.tbname)
+
+    proc = run(args.tbname)
     if args.command == 'setup':
-        proc = run(args.tbname)
         tb = get_app()
-        skip_autoconf(tb)
-        install_enigmail(tb)
-        configure_enigmail_global(tb)
+        show_menu_bar(tb)
+        skip_autoconf(tb, version)
```

Comment:
Move this call before show_menu_bar.

Reply:
Ok I thought you would have liked to do it. A moment please.